projects/plugins/jetpack/class.jetpack-connection-banner.php

```diff
@@ -251,91 +267,60 @@ public static function enqueue_connect_button_scripts() {
 	/**
 	 * Renders the new connection banner as of 4.4.0.
 	 *
-	 * @since 7.2   Copy and visual elements reduced to show the new focus of Jetpack on Security and Performance.
 	 * @since 4.4.0
+	 * @since 7.2   Copy and visual elements reduced to show the new focus of Jetpack on Security and Performance.
+	 * @since 11.1  Adjusted the banner to Emerald style
```

Comment:
```suggestion
	 * @since $$next-version$$  Adjusted the banner to Emerald style
```

You can use `$$next-version$$` in cases like this which will be automatically replaced with the correct package/plugin version number 👍